glean-core/tests/timing_distribution.rs

```diff
@@ -28,6 +28,7 @@ fn serializer_should_correctly_serialize_timing_distribution() {
         application_id: GLOBAL_APPLICATION_ID.into(),
         upload_enabled: true,
         max_events: None,
+        delay_ping_lifetime_io: false,
```

Comment:
Woha, configuration is repeated, always in the same way, across all the test files. I wonder if there's a way to share this and to create a `create_test_configuration` test-only function. That's for a follow-up PR, @brizental do you mind filing a bug for it?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1601209